src/main/java/space/obminyashka/items_exchange/util/ResponseMessagesHandler.java

```diff
@@ -72,8 +72,13 @@ public class PositiveMessage {
         public final String CHANGED_USER_INFO = "changed.user.info";
         public final String DELETE_ACCOUNT = "account.self.delete.request";
         public final String ACCOUNT_ACTIVE_AGAIN = "account.made.active.again";
-        public final String EMAIL_REGISTRATION_HEADER = "registration.header";
-        public final String EMAIL_REGISTRATION_BODY = "registration.body";
+        public final String EMAIL_REGISTRATION_TOPIC = "registration.topic";
```

Comment:
You can extract all new Registration messages into a new RegistrationMessage and that would be more informal and useful to all